tests/resources/array_config_test.yml

```diff
@@ -1,18 +1,20 @@
-site: North
-layout_name: test-layout
-model_version: Prod5
-default:
-    LSTN: 'design'
-    MSTN: 'design'
-LSTN-01:
-    name: "01"
-LSTN-02:
-    name: "02"
-LSTN-03:
-    name: "03"
-LSTN-04:
-    name: "04"
-MSTN-05:
-    name: 'design'
-    fadc_pulse_shape: Pulse_template_nectarCam_17042020-noshift.dat
-    discriminator_pulse_shape: Pulse_template_nectarCam_17042020-noshift.dat
+common:
+  site: North
+  layout_name: test-layout
+array:
+  model_version: Prod5
+  default:
+      LSTN: 'design'
+      MSTN: 'design'
+  LSTN-01:
+      name: "01"
+  LSTN-02:
+      name: "02"
+  LSTN-03:
+      name: "03"
+  LSTN-04:
+      name: "04"
```

Comment:
I thought we would solve already the issue with the name, which is given twice here (as we spoke last week). Is there a plan to do that ? And why are the names different, i.e. "LSTN-01" is "01" and not "LSTN-01"?

Reply:
As mentioned above: array config needs a rework (not as part of the PR).